ussp-api/utm-adapter.yaml

```diff
@@ -158,3 +274,425 @@ components:
           items:
             $ref: '#/components/schemas/RIDCapability'
           default: []
+   
+    PutFlightDeclarationRequest: 
+      type: object
+      properties:           
+        type_of_operation: 
+          enum:
+          - 1
+          - 2
+          - 3
+          type: integer
+          description: >
+            Set the activity type and the airspace
+             * `1` - VLOS
+             * `2` - BVLOS
+             * `3` - CREWED
+        submitted_by:
+          description: Email address of person submitting this declaration
+          type: string
+          format: email
+        originating_party:
+          description: Set the name of the flight declaration
+          type: string
+          example: Flight 1023
+        start_datetime:
+          description: The start date and time for the flight
+          type: string
+          format: date-time
+        end_datetime:
+          description: The end date and time for the flight
+          type: string
+          format: date-time
+        flight_declaration_geo_json: 
+          $ref: '#/components/schemas/FlightDeclarationGeoJSONFeatureCollection'
+    
+    FlightDeclarationGeoJSONFeatureCollection:
+      type: object
+      properties: 
+        type:
+          type: string
+          example: "FeatureCollection"
+        features:
+          type: array
+          items:
+            $ref: '#/components/schemas/FlightDeclarationFeatureGeoJSON'
+
+    FlightDeclarationFeatureGeoJSON:
+      type: object
+      required:
+        - type
+        - geometry
+        - properties
+      properties:
+        type:
+          type: string
+          enum:
+            - Feature
+        geometry:
+          $ref: '#/components/schemas/FlightDeclarationGeometryGeoJSON'
+
+        properties:
+          type: object
+          properties:
+            start_time:
+              type: string
+              format: date-time
+            end_time:
+              type: string
+              format: date-time
+            max_altitude:
+              type: object
+              properties:
+                metres:
+                  type: number
+                datum:
+                  type: string
+                  enum:
+                    - 'WGS84'
+            min_altitude:
+              type: object
+              properties:
+                metres:
+                  type: number
+                datum:
+                  type: string
+                  enum:
+                    - 'WGS84'
+          nullable: true
+        id:
+          oneOf:
+            - type: string
+              format: uuid
+            - type: integer
+
+    FlightDeclarationGeometryGeoJSON:
+      type: object
+      required:
+        - type
+      properties:
+        type:
+          type: string
+          enum:
```

Comment:
There can be more types based on USSP capabilities
Point , Linestring , Polygon , MultiPoint , MultiLineString , MultiPolygon and MultiGeometry

Reply:
Thank you, like we discussed in the last meeting, we will focus on Polygon / Multipolygon at this state, we can add additional geometry types at a later date.